### ISSUE TITLE
ErrorException in ImageFieldLocal.php line 23: Invalid argument supplied for foreach()

### ISSUE THREAD

**Issue by KalebClark** [sahusoftcom/eloquent-image-mutator#9]
Evening,

I get this error when I try to query or save an image. Below is the method for uploading the file. I am returning a new object because I was trying to see if returning $event after the save had issues.

Any Ideas?

Thanks! Great Package!

```
    public function upload(Request $request, $id)
    {
        $event = Event::findOrFail($id);
        $event->event_image    = $request->file('event_image');
        $event->save();

        return Event::findOrFail($id);
    }
```

**Comment by mrigankmridul** [comment]
Hey @KalebClark you are welcome :-) Could you answer 4 questions for me ?

Is the field `event_image` a text field ?
Could you send me the data updated in your field?
And the data in your /config/image.php? 
And what type of image file are you trying to upload ?

**Comment by KalebClark** [comment]
I think I figured it out while gathering the information to answer your four questions.

Turns out in my seeder, I had put '2' in the event_image text field. When the mutator got hold of that it would choke because I think it was expecting to convert whatever is in that field to json. I made the text('event_image')->nullable(), set it to null in the seeder, migrated and seeded. It works now...

Thanks for the quick reply. I am going to be using this package a bunch in future projects. Good stuff.

-Kaleb
